### ISSUE TITLE
Can Novu be installed as part of my current Express project or does it have to be a standalone service in a separate package?

### ISSUE THREAD

**Issue by codeser** [novuhq/novu#1274]
- **I'm submitting a ...**
  [ ] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [x] question about how to use this project

- **Summary**

I noticed the way to install Novu is using 

`npx novu init`

Can I instead do an npm install novu in my current Express project, save novu as a library and call it internally from my other files/functions?

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

**Comment by JakovGlavac** [comment]
Yeah sure, but if you are wondering what does init script do check the source code: https://github.com/novuhq/novu/blob/main/packages/cli/src/commands/init.ts

**Comment by scopsy** [comment]
Hi @codeser! Unfortunately, novu can only be consumed as a standalone service using the HTTP API. You can either self-host it yourself or use our cloud offering, regarding each path you will choose would love to assist setting it up 🙏

You could still install @novu/node to make the trigger calls to the Novu Platform API, but you won't be able to do that before the API is running somewhere as a standalone service.
